core/src/main/scala/org/apache/spark/storage/StorageStatusListener.scala

```diff
@@ -17,26 +17,55 @@
 
 package org.apache.spark.storage
 
+import java.util.concurrent.TimeUnit
+
+import scala.collection.JavaConversions.collectionAsScalaIterable
 import scala.collection.mutable
+import scala.language.reflectiveCalls
 
+import org.apache.spark.SparkConf
 import org.apache.spark.annotation.DeveloperApi
 import org.apache.spark.scheduler._
 
+import com.google.common.base.Ticker
+import com.google.common.cache.CacheBuilder
```

Comment:
import ordering -- these go above the o.a.spark imports, with a linebreak in between.  See https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports